src/workerd/jsg/jsg.c++

```diff
@@ -9,6 +9,8 @@
 #include <workerd/jsg/util.h>
 #include <workerd/util/thread-scopes.h>
 
+#include <kj/exception.h>
```

Comment:
It's not clear why this additional include is needed?

Reply:
Ooops.